.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Lint and validate OpenAPI specs
+on:
+  - push
+  - pull_request_target
+jobs:
+  lint:
+    name: Lint OpenAPI definition
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out head branch
+        uses: actions/checkout@v4
+      - name: Run OpenAPI Lint Action
+        uses: nwestfall/openapi-action@v1.0.2
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Isn't this risky, together with `pull_request_target` above? If I understand this correctly, people creating a PR from a fork might be able to read the `GITHUB_TOKEN` and could mess with the repo as per the permissions the token has.

On the other hand, with the current configuration, this job wouldn't run if triggered from a fork (which is our usual workflow). Perhaps we should check whether we can lint the workflow without the secret and then use `pull_request` instead of `pull_request_target`? It seems to me that a linting check shouldn't need to require specific permissions (just like the `diff` and `validate` jobs don't need it).

Reply:
Testing a new plugin for validation `mhiew/redoc-lint-github-action@v4`